Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Hydroponics/seeds.yml

```
@@ -0,0 +1,19 @@
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/pineapple.rsi
+
+- type: entity
+  parent: SeedBase
+  name: packet of plasmapple seeds
+  id: PlasmappleSeeds
+  components:
+    - type: Seed
+      seedId: plasmapple
+    - type: Sprite
+      sprite: _Goobstation/Objects/Specific/Hydroponics/plasmapple.rsi
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)